cluster/charts/crossplane/README.md

```diff
@@ -0,0 +1,270 @@
+# Install Crossplane
```

Comment:
Would it make sense for this document to link to our existing documentation on these topics, rather than repeating it? Doing so may reduce our documentation maintenance burden.

Reply:
@displague any thoughts here? Looks like we can get this merged after resolving this comment :)

Reply:
@negz @hasheddan Sorry for missing this.

https://hub.helm.sh/charts/jetstack/cert-manager is an example of what this README.md looks like rendered.  If think we are addressing the wrong audiences at the wrong places if we defer to our documentation in Helm hosted locations.

As an alternative, our documentation could link to Helm Hub as the best source for detailed Helm usage instructions.  Helm Hub's content is rendered from this chart published file (which is lightly modified from docs/install-crossplane.md to remove the front-matter).

```patch
diff docs/install-crossplane.md cluster/charts/crossplane/README.md
1,6d0
< ---
< title: Install Crossplane
< toc: true
< weight: 220
< indent: true
< ---
```

Reply:
> I think we are addressing the wrong audiences at the wrong places if we defer to our documentation in Helm hosted locations.

@displague are you saying you prefer not to link to the crossplane docs in the chart README? If so, it looks like this is the same as our install page on the docs, so maybe we just setup automation to sync it here during build? As is, these instructions are already out of date from our docs :/

Reply:
> setup automation to sync it here during build
That would be a good solution.  I considered using a symlink, but symlinks in git repos can be troublesome in certain settings and the contents of the files are slightly different.

Reply:
Would you be ok with this file being sync'd via the Makefile's `manifests.annotate`?
```
sed '1 { /^---/ { :a N; /\n---/! ba; d} }' $(SOURCE_DOCS_DIR)/install-crossplane.md > $(CROSSPLAN E_CHART_DIR)/README.md
```

As we do now for chart files, we would have to commit changes to this file as needed.

Reply:
That seems reasonable to me. I would want a check in place in CI to make sure these two files were in sync, seems like we could just run your command as part of [`check-diff`](https://github.com/crossplaneio/crossplane/blob/1b8c665bb2511f47d089444234d8762b2661f6d6/Makefile#L133).

Reply:
`check-diff` would detect any drift as-is, so long as a make `manifests` is run during CI (part of `reviewable`, I think).

Reply:
Sounds great!

Reply:
The file is generated in the latest push, but the sed command isn't working on OSX.

Reply:
`sed '/^---/,/^---/d'` does the trick with both BSD and GNU sed.